### PR TITLE
Add slidedeck for August 2017

### PR DESCRIPTION
**Pull request by lcobucci**
Preview: https://lcobucci.github.io/monthly-slidedeck/2017-August.html#/

### CODE REVIEW

**Comment by lcobucci** [comment]
We still need to add the joind.in link

**Comment by lcobucci** [comment]
joind.in link added, just waiting for your review 😉

File: 2017-August.html [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+                            <img id="hoster-small" class="hoster" src="" alt="">
+                        </div>
+
+                        <div>
+                            <ul class="pointless">
+                                <li>19h00 - 19h30: Welcome &amp; Drinks</li>
+                                <li>19h30 - 19h45: Opening</li>
+                                <li>19h45 - 20h30: Talk</li>
+                                <li>20h30 - 22h00: Raffle &amp; Social</li>
+                            </ul>
```
Comment (dennisdegreef) [inline 2017-August.html]: https://www.meetup.com/AmsterdamPHP/events/240083692/ sais:
```
19:00 - Doors open  
19:30 - Talk  
20:30 - Raffles, and announcements  
20:45 - Social, drinks and Pizza
```

**Comment by dennisdegreef** [comment]
LGTM :+1:

**Pull request merged**